src/bash/delete_trash

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
We don't need this if you'll use
hadoop fs -rm -r -skipTrash /hdfs_path command all the time.